udplog/test/test_kafka.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Ralph Meijer.
```

Comment:
It was copy-pasted from *test_redis.py* file. What do you want me to put
here?

On Wed, Nov 11, 2015 at 2:07 PM, Christopher Zorn <notifications@github.com>
wrote:

> In udplog/test/test_kafka.py
> <https://github.com/mochi/udplog/pull/10#discussion_r44593730>:
>
> > @@ -0,0 +1,94 @@
> > +# Copyright (c) Ralph Meijer.
>
> Change please
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mochi/udplog/pull/10/files#r44593730>.
>